### PR TITLE
Don't let changes sidebar get too small

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
Adds a minimum width (up from the default of `150px`) for `RepositoryView`.

### Before
before

### After
after

I'd love to know what @desktop/design thoughts are. I felt the default of `150px` was a bit too cramped. Is setting a min width of `225px` too much?

### CODE REVIEW

**Comment by agisilaos** [comment]
That's reasonable. @iAmWillShepherd I'm thinking though what if the path of the file is bigger than the min width of `225px`. It will look simillar like that (a bit shrinked): 

screen shot 2018-02-23 at 13 30 52

Note: 
Take that screenshot as an example of what it's gonna look like in the `Changes` tab

**Comment by niik** [comment]
> I'd love to know what @desktop/design thoughts are. I felt the default of 150px was a bit too cramped. Is setting a min width of 225px too much?

It is cramped but I don't think we should be restrictive here. We initially had a [minimum width of 200px](https://github.com/desktop/desktop/commit/f2c37777acbec8757dee55e80b022976431c7a75) but I decided to decrease it to 150px.

It's not necessarily about whether the sidebar looks good in 150px but rather about making more room for the diff temporarily. I think the question we should be asking ourselves when we look at resizable widths is the minimum width where all components can still be used. In this case the limiting factor is whether we can fit the 'Changes' and 'History' tabs in the width needed and we can in 150px.

So in summary I don't think we should increase the minimum width until we have components in the sidebar which become unusable at the minimum width.

**Comment by j-f1** [comment]
What about “snapping” it to 225px if you resize it to somewhere between 220px and 230px? This would allow us to suggest a minimum width, while still allowing you to make it narrower if you need to.

**Comment by niik** [comment]
> What about “snapping” it to 225px if you resize it to somewhere between 220px and 230px? This would allow us to suggest a minimum width, while still allowing you to make it narrower if you need to.

You can get back to the default width by double clicking on the resizable. I don't necessarily have anything against snapping as well other than that I don't think there's enough precedence for that kind of interaction. I suspect most people will leave the sidebar at its starting position which is what we've determined to be a good look for the application. Ultimately I'd want to work from a clear use case and example of this being an actual pain point before we look at solutions.

**Comment by iAmWillShepherd** [comment]
> So in summary I don't think we should increase the minimum width until we have components in the sidebar which become unusable at the minimum width.

Fair enough 👍

**Comment by iAmWillShepherd** [comment]
> I'm thinking though what if the path of the file is bigger than the min width of 225px. It will look simillar like that (a bit shrinked):

@agisilaos It would likely still looked cramped, but to a lesser degree which is what I was going for really. one of the things that actually spurred me to make this change is my commit messages typically are more than 3 words, so I figured why not make it wide enough for at least 6 words?! 😆

**Comment by agisilaos** [comment]
@iAmWillShepherd So damn true, same here! 😆
